testkit/src/main/scala/org/bitcoins/testkit/wallet/FundWalletUtil.scala

```diff
@@ -68,11 +69,7 @@ trait FundWalletUtil {
       hashes <- bitcoind.getNewAddress.flatMap(bitcoind.generateToAddress(6, _))
     } yield (tx, hashes.head)
 
-    val fundedWalletF =
```

Comment:
This was the cause of a lot of problems in the node project, our wallet would have transactions without having the internal chainstate/wallet state (`wallet.getSyncHeight()`) did not match what the bitcoind has.